interface/src/ui/PreferencesDialog.cpp

```diff
@@ -257,11 +257,16 @@ void setupPreferences() {
     }
 
     {
-        auto getter = []()->bool { return !Menu::getInstance()->isOptionChecked(MenuOption::DisableCrashLogger); };
-        auto setter = [](bool value) { Menu::getInstance()->setIsOptionChecked(MenuOption::DisableCrashLogger, !value); };
-        preferences->addPreference(new CheckPreference("Privacy", "Send crashes - Vircadia uses information provided by your "
-                                "client to improve the product through crash reports. By allowing Vircadia to collect "
-                                "this information you are helping to improve the product. ", getter, setter));
+        auto getter = []()->bool { return
+            Menu::getInstance()->isOptionChecked(MenuOption::GenerateAndSubmitCrashReports)
+            && UserActivityLogger::getInstance().isCrashMonitorEnabled()	// Reset to unchecked if feature is disabled so users aren't locked in.
```

Comment:
Resetting to unchecked seems like a bad idea.
If someone enables crash reporting they will most likely have it enabled wherever possible. If they then run a build that doesn't come with crash reporting, it will disable their setting without them knowing. We already have fairly little people that turn the option on, so I would hate so have it turn off for them automatically somehow.

Reply:
Nevermind just saw your comment.

Reply:
This only resets the checkbox, not the setting. It may reset the setting if the user saves with the checkbox unchecked or it may not. The taskbar checkbox indicates the true value of the setting. Do you have a better suggestion? I'm going to update my previous comment on this PR to correct the problem statement.